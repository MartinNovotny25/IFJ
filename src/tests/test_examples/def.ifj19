""" funkce
DEF ID ( <arg-list> ) : EOL INDENT <state> EOL <st-list> <return> <st-list> DEDENT """

def main():
    a = 1
    print(a)

def foo():
    a = 1 + 2 + a
    print(a)

def foo1(a, b, c):
    a = 1 + b + c
    print(a)

x = 1+2
main()
foo()
foo1(1)
foo1(x)
input(i)
