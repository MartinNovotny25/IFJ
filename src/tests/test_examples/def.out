TOKEN(EOL)
TOKEN(EOL)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, 1)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, 1)
TOKEN(+)
TOKEN(INTEGER, 2)
TOKEN(+)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, 1)
TOKEN(+)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, 1)
TOKEN(+)
TOKEN(INTEGER, 2)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(INTEGER, 1)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOF)
